src/path-to-coordinates.js

```diff
@@ -3,15 +3,20 @@ import { range } from 'lodash';
 function pathToCoords(path, scale, numPoints, translateX, translateY) {
   const length = path.getTotalLength();
   const getRange = range(numPoints);
-
+  const id = path.id ? path.id : null;
   // Always include the max value in the range.
   // This is helpful for detecting closed polygons vs lines
   getRange.push(numPoints);
 
-  return getRange.map(function(i) {
-    const point = path.getPointAtLength(length * i / numPoints);
-    return [point.x * scale + translateX, point.y * scale + translateY];
-  });
+  return {
+    id,
```

Comment:
Can we just pass `path` here? 

```js
return {
  ...path,
  coords: [ /*... */ ]
}
```

Reply:
You want to pass the whole `path` here? Why? For now, we only need the id. If we want to handle more attribute in the future, we can but right now we would be passing something like this: `<path id="SC" d="M594.979 316.383l8.844 5.106 8.845-5.106 8.844 5.106 8.845-5.106 8.844 5.106v10.213l-8.844 5.107v10.212l-8.845 5.107v10.212l-8.844 5.107-8.845-5.107-8.844 5.107-8.845-5.107v-10.212l-8.844-5.107v-10.212l-8.845-5.107v-10.213l8.845-5.106 8.844 5.106z"></path>`

Reply:
Oh I see. I thought this was passing a regular object. Yeah, I was thinking if we wanted to include more in the future we wouldn't need to adjust this code each time. It also saved from [the](https://github.com/mapbox/svg-to-geojson/blob/71ad9131dea3ef3879caf09d9edbc195848f722a/src/components/app.js#L76-L78) [two](https://github.com/mapbox/svg-to-geojson/blob/94380336794d8baec68262576127c0b2cb5c218f/src/path-to-coordinates.js#L6) conditional checks you have here.

Reply:
Can I resolve for now?